utils/ccloud_library.sh

```diff
@@ -356,7 +367,8 @@ function ccloud::maybe_create_and_use_cluster() {
 function ccloud::create_service_account() {
   SERVICE_NAME=$1
 
-  OUTPUT=$(ccloud service-account create $SERVICE_NAME --description $SERVICE_NAME  -o json)
+  CCLOUD_EMAIL=$(ccloud prompt -f '%u')
+  OUTPUT=$(ccloud service-account create $SERVICE_NAME --description "SA for $EXAMPLE run by $CCLOUD_EMAIL"  -o json)
```

Comment:
I'm curious if we might benefit from using the description fields to put in more parsable text? Also do we need the word "SA" when it's already attached to a Service Account?

FYC instead of:

"SA for example run by user"

"user:example" would be easier to parse if needed by a machine and I'm not sure that the other wording does all that much for the human reader as they will see the description in the context of listing of service accounts.

Just my opinion...

Reply:
@rspurgeon `SA for ` language was intentionally borrowed from the ksqlDB description when a new app is provisioned.  We don't have to be consistent with that, but that was the starting point.  @rspurgeon @javabrett preferences for consistency ksqlDB over parsability?